include/dt-bindings/interrupt-controller/intel-ioapic.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DT_BINDING_IRQ_H
+#define __DT_BINDING_IRQ_H
+
+#define	IRQ_TYPE_LEVEL		0x00008000
+#define	IRQ_TYPE_EDGE		0x00000000
+#define	IRQ_TYPE_LOW		0x00002000
+#define	IRQ_TYPE_HIGH		0x00000000
+#define EDGE_RISING		(IRQ_TYPE_EDGE | IRQ_TYPE_HIGH)
```

Comment:
all these defines should be prefixed with IRQ_TYPE_

Reply:
ok will do it